src/uct/ib/base/ib_iface.c

```diff
@@ -156,7 +156,20 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    "IB Hop limit / RoCEv2 Time to Live. Should be between 0 and 255.\n",
    ucs_offsetof(uct_ib_iface_config_t, hop_limit), UCS_CONFIG_TYPE_UINT},
 
-  {"LID_PATH_BITS", "0-17",
+  {"NUM_PATHS", "auto",
+   "Number of paths to expose for the interface. 'auto' means use "
+   UCS_PP_MAKE_STRING(UCT_IB_DEV_MAX_PORTS) " paths\n"
+   "for RoCE LAG devices and 2^LMC paths for InfiniBand fabric. In some\n"
+   "cases using multiple paths can get higher bandwidth overall.",
+   ucs_offsetof(uct_ib_iface_config_t, num_paths), UCS_CONFIG_TYPE_ULUNITS},
+
+  {"ROCE_PATH_FACTOR", "1",
```

Comment:
why it needs to be configurable?

Reply:
the value we want to use can depend on fabric switch model/configuration/wiring in case of RoCE